src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.engine.backend
+
+import cromwell.binding.values.WdlValue
+import cromwell.binding.{Call, WdlStandardLibraryFunctions, WorkflowDescriptor}
+
+import scala.util.Try
+
+trait BackendCall {
```

Comment:
Can you put a comment here describing what this, why this is and what all the pieces are for? Like the other stuff before, this would be for the benefit of an implementer of a new backend who might even be a complete outsider